code/datums/keybinding/xeno.dm

```diff
@@ -249,6 +249,14 @@
 	description = ""
 	keybind_signal = COMSIG_XENOABILITY_REAGENT_SLASH
 
+/*
+/datum/keybinding/xeno/defile
+	name = "defile"
+	full_name = "Defiler: Defile"
+	description = "Purges xeno toxins in exchange for dealing toxin damage and generating toxic sanguinal smoke."
+	keybind_signal = COMSIG_XENOABILITY_DEFILE
+*/
```

Comment:
as of https://github.com/tgstation/TerraGov-Marine-Corps/pull/7182 you can supposedly remove keybinds so you dont need to comment them for TM anymore.